### PR TITLE
Add kube-score to index

### PR DESCRIPTION
**Pull request by zegl**

### CODE REVIEW

File: plugins/score.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    uri: https://github.com/zegl/kube-score/releases/download/v1.5.1/kube-score_1.5.1_linux_amd64.tar.gz
+    sha256: "495daa3be08a8945c7926eb91f19b5f6c1b4fe105f1ef67450d37e7aeccf0b05"
+    files:
+    - from: "kube-score"
+      to: .
+    - from: "LICENSE"
+      to: .
+    bin: kube-score
+
+  shortDescription: Kubernetes object analysis with recommendations for improved reliability and security.
```
Comment (ahmetb) [inline plugins/score.yaml]: This is too long 

We need about half the current length   :)

**Comment by ahmetb** [comment]
Only one comment. The rest looks good.

**Comment by zegl** [comment]
> Only one comment. The rest looks good.

Thanks for the review 👍 , I've pushed a new commit with a shorter shortDescription.

**Comment by corneliusweig** [comment]
/lgtm
/approve
Welcome to krew index!

**Pull request merged**